docs/neo-n3/dapp_development/introduction.md

```diff
@@ -1,37 +1,35 @@
-with a single method call, or construct a transaction from scratch using the `TransactionBuilder`.
-
-The neow3j SDK is divided into two modules `io.neow3j:core` and `io.neow3j:contract` that separate the lower level core
-functionalities, like signing, from more abstract functionality, like building contract-specific transactions. The
-_core_ module is a dependency of the _contract_ module, so if you add _contract_ as a dependency to your project you
-will get the _core_ module as well.
-
-**`io.neow3j:core`** contains
-  - Basic definitions and enums.
-  - Utility methods of which the most notable classes are, e.g., for handling hex strings. 
-  - Cryptographic methods related to key pairs (class `ECKeyPair`) and signatures (class `Sign`).
-  - Wallet and account management, i.e., the handling of key pairs or reading from and writing to NEP-6 wallet files.
-  - Interaction with the Neo blockchain, i.e., communication via JSON-RPC with a Neo node. 
-  - The `Transaction` and `ScriptBuilder` classes that are fundamental for constructing contract invocations of any
-    kind.
-
-**`io.neow3j:contract`** contains
-- Functionality related to smart contracts.
-- Classes that allow easy interaction with native contracts.
-- Classes that represent and allow easy interaction with contracts following a certain standard, like the NEP-17 token
-  standard.
-- for interfaces like fungible or non-fungible token contracts and native contracts that support building invocation
-  scripts to create a transaction.
-
-The following sections explain the SDK's capabilities based on common concepts and activities used in dApp
-development.
+In our definition, dApp development includes the creation of systems that interact and are based on a blockchain. Although smart contracts are also a component of a dApp, we distinguish between contract development and dApp development because neow3j's libraries cleanly separate these two activities. In this section, we introduce the neow3j SDK, which focuses on dApp development.
+
+The neow3j SDK aims to offer a high-level abstraction layer that frees developers from handling the technical intricacies of the Neo blockchain. Simultaneously, it provides the capability for more detailed configuration in scenarios requiring greater control.
+
+For instance, you can utilize the `NeoToken` class to easily create a NEO transfer with a single method call, or build a transaction from the ground up using the `TransactionBuilder`.
+
+The neow3j SDK is divided into two modules: `io.neow3j:core` and `io.neow3j:contract`. These modules separate lower-level core functionalities, such as signing, from more abstract functionalities, such as building contract-specific transactions. The `core` module is a dependency of the `contract` module. Therefore, if you add `contract` as a dependency to your project, you will automatically include the `core` module as well.
```

Comment:
I'm not so sure if this part of the docs was affected by https://github.com/neow3j/neow3j/pull/1050.

@thedanielmark, my suggestion is: create an issue here in `neow3j-docs` to investigate that (assigned it to you), and add to our AxLabs board to tackle in the upcoming sprint.